lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
This line in PubCentral is crazy. We should 'use strict' everywhere!! We shouldn't assume the ESLint file is correct, that file was just thrown inthere and I'm not sure by who.

Note 'use strict' is an important one because it changes the JS engine behavior and makes it a bit safer. So we should use it.

That being said @e2tha-e caveat is correct. We shouldn't just dump it in the top of files. I think in node that's fine if you put it inside a module block but not in the global like that.